content/en/posts/css/introduction-to-cssnext/index.md

```diff
@@ -0,0 +1,337 @@
+}
+```
+The code above will be transformed into...
+
+```css
+.class {
+  background-color: rgb(43, 136, 230);
+  color: rgba(73, 86, 15, 0.65);
+  border-top-color: rgb(181, 201, 222);
+  border-right-color: rgb(3, 45, 87);
+  border-bottom-color: rgb(213, 231, 250);
+  border-left-color: rgb(11, 34, 58);
+}
+```
+
+**cssnext** also offers the following color-related features.
+
+#### hwb()
+
+From the [specifications](http://dev.w3.org/csswg/css-color/#the-hwb-notation), HWB (Hue-Whiteness-Blackness) is similar to HSL but easier for humans to work with.
+
+```css
+.title {
+  color: hwb(125, 32%, 47%);
+}
+```
+
+Output:
+
+```css
+.title {
+  color: rgb(33, 135, 42);
+}
+```
+
+#### gray()
+
+Grays are [so cool](http://dev.w3.org/csswg/css-color/#grays) they have a function of their own.
+
+```css
+.section {
+  background-color: gray(120, 50%);
+  border-color: gray(17%, 25%);
+}
+```
+
+This will output:
+
+```css
+.section {
+  background-color: rgba(120, 120, 120, 0.5);
+  border-color: rgba(43, 43, 43, 0.25);
+}
+```
+
+#### #rrggbbaa
+
+**cssnext** transforms the [hexadecimal notations](http://dev.w3.org/csswg/css-color/#hex-notation) #RRGGBBAA and #RGBA into rgba().
+
+```css
+body {
+  color: #5c69;
+  background-color: #C73D5C59;
+}
+```
+
+Output:
+
+```css
+body {
+  color: rgba(85, 204, 102, 0.6);
+  background-color: rgba(199, 61, 92, 0.34902);
+}
+```
+
+#### rebeccapurple
+
+Simply transforms the color [`rebeccapurple`](https://github.com/postcss/postcss-color-rebeccapurple#why-this-plugin-) into `rgb(102, 51, 153)`.
+
+### filter properties
+
+A whole _new_ world of [image modifications](http://www.w3.org/TR/filter-effects/)!
+
+```css
+.awesome-Image {
+  filter: sepia(.7) hue-rotate(23deg);
+}
+
+.awesome-Picture {
+  filter: blur(8px);
+}
+```
+
+Will be transformed into:
+
+```css
+.awesome-Image {
+  filter: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg"><filter id="filter"><feColorMatrix type="matrix" color-interpolation-filters="sRGB" values="0.5751000000000001 0.5383 0.1323 0 0 0.24429999999999996 0.7802000000000001 0.11760000000000001 0 0 0.1904 0.3738 0.39170000000000005 0 0 0 0 0 1 0" /><feColorMatrix type="hueRotate" color-interpolation-filters="sRGB" values="23" /></filter></svg>#filter');
+  -webkit-filter: sepia(.7) hue-rotate(23deg);
+          filter: sepia(.7) hue-rotate(23deg);
+}
+
+.awesome-Picture {
+  filter: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg"><filter id="filter"><feGaussianBlur stdDeviation="8" /></filter></svg>#filter');
+  -webkit-filter: blur(8px);
+          filter: blur(8px);
+}
+```
+
+### rem units
+
+Nothing extraordinary here, it generates **pixel fallback for rem units**.
+Oh, come on! Do we really need an example for this one? Alright, here you go!
+
+```css
+.section-Highlight {
+  font-size: 2.5rem;
+}
+```
+Output:
+
+```css
+.section-Highlight {
+  font-size: 40px;
+  font-size: 2.5rem;
+}
+```
```

Comment:
like for filters, I would add a comment about browser scope (some people would think rem to px is a useless transformation these days)